src/libraries/Common/tests/System/Net/Http/HttpClientHandlerTest.Authentication.cs

```diff
@@ -203,7 +203,7 @@ public static IEnumerable<object[]> Authentication_TestData()
         {
             yield return new object[] { "Basic realm=\"testrealm\"", true };
             yield return new object[] { "Basic ", true };
-            yield return new object[] { "Basic realm=withoutquotes", true };
+            yield return new object[] { "Basic realm=PLACEHOLDERwithoutquotes", true };
```

Comment:
credscan picked up on the "realm" part here? but not on line 204? It is strange...